snapcraft/ovn-central.env

```diff
@@ -0,0 +1,21 @@
+# Set of environment variables used by OVN Central services
+
+# Load runtime OVN environment variables
+. "${SNAP_COMMON}/data/ovn.env"
+
+# Setup directories
+export OVN_DBDIR="${SNAP_COMMON}/data/central/db"
+export OVN_LOGDIR="${SNAP_COMMON}/logs"
+export OVN_RUNDIR="${SNAP_COMMON}/run/ovn"
+export OVN_PKGDATADIR="${SNAP}/share/ovn"
+export OVN_SYSCONFDIR="${SNAP}/etc"
+export OVN_PKIDIR="${SNAP_COMMON}/data/pki"
+
+# Disable some commands
+mkdir -p "${OVN_RUNDIR}/bin/"
+for i in install plymouth sudo systemctl; do
+    [ -e "${OVN_RUNDIR}/bin/${i}" ] && continue
+
+    ln -s "/bin/true" "${OVN_RUNDIR}/bin/${i}"
+done
```

Comment:
While I understand how these ended up in this file, it's probably not the correct place to host them after the split.

Having three services starting independently attempting to create this might lead to a race conditions and spurious errors. (And having an sourceable "environment" file performing actions behind your back comes as an element of surprise.)

Would it make sense to move this to either the install hook or as part of the startup of the `daemon` service?

Reply:
I'd rather not have them in a sourceable `.env` file.

Speaking of who needs this, I honestly do not think this is related to any of the central services at all, I do not find any evidence of the `ovn-ctl`, `ovs-ctl` or `ovs-lib` scripts making calls to these binaries. So I wonder what was the reason for the original introduction of this, habits of the original author?

Reply:
I'd be in favor of removing this then. The symlinks do not even have effect without the very specific `{$OVN_RUNDIR}/bin/` added to path, and that happens only here.